src/profile-logic/zip-files.js

```diff
@@ -195,8 +195,17 @@ export class ZipFileTree {
           encodeURIComponent(this._zipFileTable.path[zipTableIndex]);
       }
 
+      let name = this._zipFileTable.partName[zipTableIndex];
+      const EXTENSIONS_TO_OMIT = ['.json', '.json.gz'];
```

Comment:
oh actually it looks like we don't support .json.gz files inside a zip file, it just doesn't work... So it makes no sense to remove it . Can you change it and just keep `".json"`? I think it makes sense to keep an array in case we want to add some more later. (Or otherwise use the Regexp like mentioned in the other comment)

And maybe eventually we'll need to support .json.gz inside a zip...

Reply:
Done!